src/liballoc/lib.rs

```diff
@@ -69,7 +69,10 @@
        issue_tracker_base_url = "https://github.com/rust-lang/rust/issues/",
        test(no_crate_inject, attr(allow(unused_variables), deny(warnings))))]
 #![no_std]
-#![needs_allocator]
+
+// Don't inject an allocator in stage0 since we explicitly use alloc_system, jemalloc
+// being possibly unavailable or disallowed
+#![cfg_attr(not(stage0), needs_allocator)]
```

Comment:
Ah in that case the `make install` issue was likely unrelated and connected to some other series of events. I believe that if this is reverted back to as-it-was it should bootstrap A-OK